vscode/webviews/Components/platform/SelectList.tsx

```diff
@@ -0,0 +1,77 @@
+import classNames from 'classnames'
+import { type FunctionComponent, type ReactNode, useCallback, useState } from 'react'
+import { FLOATING_WIDGET_CLASS_NAME } from './FloatingWidget'
+import styles from './SelectList.module.css'
+
+type Value = string
+
+export interface SelectListOption {
+    value: Value | undefined
+    title: string | ReactNode
+    disabled?: boolean
+}
+
+export const SelectList: FunctionComponent<{
+    options: SelectListOption[]
+    value: string | undefined
+    onChange: (value: Value | undefined, close: boolean) => void
+    className?: string
+}> = ({ options, value, onChange, className }) => {
+    const [isKeyDown, setIsKeyDown] = useState(false)
+
+    const suppressBlur = useCallback((e: React.MouseEvent<HTMLElement>): void => {
+        e.stopPropagation()
+        e.preventDefault()
+    }, [])
+
+    return (
+        <div className={classNames(styles.container, FLOATING_WIDGET_CLASS_NAME, className)}>
+            <ul
+                className={styles.list}
+                role="radiogroup"
+                onKeyDown={e => {
+                    setIsKeyDown(true)
+                    if (e.key === 'Enter') {
```

Comment:
Native select boxes can also be changed via space:

```suggestion
                    if (e.key === 'Enter' || e.key === ' ' ) {
```